### DISCUSSION TITLE
Keep Tooltip Visible At Last Position

### DISCUSSION THREAD

**Discussion by ghost** [recharts/recharts#3396] [Q&A] [answered]
Hi guys,

Is there a way to keep the tooltip visible at the last mouse position before it leaves the graph?

**Comment by ckifer** [comment]
Not currently https://github.com/recharts/recharts/issues/855

**Reply by ghost** [reply]
Would it be possible to somehow keep the isTooltipActive as true at the last activeTooltipIndex? Or disable setting the isTooltipActive to false onMouseLeave?

**Reply by ckifer** [reply]
I don't think so at this point

**Reply by Yilun-Sun** [reply, marked as answer]
Hi @um-cle , please see this comment.
https://github.com/recharts/recharts/issues/855#issuecomment-1502645268

**Reply by ghost** [reply]
Thank you!
